src/global/CMakeLists.txt

```diff
@@ -17,3 +17,8 @@ target_link_libraries(global-static common)
 add_library(global STATIC
   $<TARGET_OBJECTS:libglobal_objs>)
 target_link_libraries(global ceph-common ${EXTRALIBS})
+
+#global that does not include ceph-common
+add_library(global-raw STATIC
```

Comment:
Suggest to edit the commit message to add a reason for having that target (not just what it is)